shell/platform/darwin/ios/framework/Source/SemanticsObjectTest.mm

```diff
@@ -12,6 +12,12 @@
 
 const CGRect kScreenSize = CGRectMake(0, 0, 600, 800);
 
+// copied from clang document: https://clang-analyzer.llvm.org/faq.html#unlocalized_string
```

Comment:
Couldn't we just use NSLocalizedString everywhere? It's a test, so it's not like that's going to affect a shipped binary.